packages/primes/primes.1.0/opam

```diff
@@ -0,0 +1,30 @@
+
+opam-version: "1.2"
+name: "primes"
+version: "1.0"
+maintainer: "Kit Freddura <kitfreddura@gmail.com>"
+authors: "Kit Freddura <kitfreddura@gmail.com>"
+license: "MIT License"
+dev-repo: "https://github.com/KitFreddura/OCaml-Primes.git"
+homepage: "https://github.com/KitFreddura/OCaml-Primes"
+bug-reports: "None yet"
+build: [
+  ["mkdir" "%{prefix}%/lib/primes"]
+]
+ocaml-version: [>= "4.01"]
+install: [
+["ocamlfind" "ocamlc" "-c" "-linkpkg" "-package" "zarith" "primes.mli"]
+["ocamlfind" "ocamlc" "-thread" "-o" "primes.cma" "-linkpkg" "-package" "zarith,core,gen" "primes.ml"]
+["cp" "primes.cmo" "%{prefix}%/lib/primes/"]
+["cp" "primes.cmi" "%{prefix}%/lib/primes/"]
+["cp" "primes.cma" "%{prefix}%/lib/primes/"]
+["cp" "primes.mli" "%{prefix}%/lib/primes/"]
+["cp" "META" "%{prefix}%/lib/primes/"]
+]
+remove: ["ocamlfind" "remove" "primes"]
+depends: [
+  "ocamlfind" {build}
+  "zarith" {build}
+  "core" {build}
+  "gen" {build}
+]
```

Comment:
Are these really all build-only dependencies?  If your library relies on any of them at run time then you should remove `{build}`.  It appears that only `ocamlfind` is a build-only dependency.